packages/menu/menu-item.md

```diff
@@ -57,7 +57,33 @@ Content assigned to the `icon` slot will be placed at the beginning of the `<sp-
 </sp-menu>
 ```
 
-### Description slot
+##### Icon management
```

Comment:
This renders super small for me on the docs site. Could we bump it up by 1 (and same for the Icon header above?) 
```suggestion
#### Icon management
```